packages/datadog-plugin-amqp10/src/consumer.js

```diff
@@ -11,8 +11,8 @@ class Amqp10ConsumerPlugin extends ConsumerPlugin {
     const source = getShortName(link)
     const address = getAddress(link)
 
-    this.startSpan('amqp.receive', {
-      service: this.config.service || `${this.tracer._service}-amqp`,
+    this.startSpan(this.operationName(), {
```

Comment:
Every span from every plugin will need this. Should it just be directly in `startSpan`?

Reply:
Eventually yes, but I don't think we can generalize it until every plugin has its naming data. I can generalize it for `Consumer` and `Producer` plugins though, since they're now all covered. Until we tackle all `Client` plugins, we'll still have this for the control plane part of kafka and amqplib.